### PR TITLE
wrap-promise fix

### PR DESCRIPTION
**Pull request by targunp**
# Summary

Fix for issue #102 

# Checklist

- [x] Added changelog entry
- [x] Ran unit tests (`npm test`)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "semver": "5.1.0",
     "underscore": "1.8.3",
     "xml2js": "0.1.13",
-    "wrap-promise": "braintree/wrap-promise#1.1.0"
+    "@braintree/wrap-promise": "git+https://github.com/braintree/wrap-promise.git"
```
Comment (crookedneighbor) [inline package.json]: The version for this should just be: `1.1.1`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "braintree",
-  "version": "2.1.1",
+  "version": "2.1.2",
```
Comment (crookedneighbor) [inline package.json]: Please leave this as 2.1.1 until we're ready to publish a release.

Comment (crookedneighbor) [inline package.json]: We'll bump the version when we publish it.

**Comment by targunp** [comment]
I am using node v8: https://nodejs.org/en/

**Comment by targunp** [comment]
Cool, thanks

**Comment by crookedneighbor** [comment]
Thanks for doing this. It will go out in the next release.

**Pull request merged**